### ISSUE TITLE
Fracture of the lower leg

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5922]
**Preferred term label:** Fracture of the lower leg

**Synonyms** Lower extremity fracture; Lower limb fracture

**Definition (free text, please give PubMed ID)** Bone fracture in the tibia, fibula, or ankle., PMID:20109361

**Parent term (use hpo.jax.org/app)** Abnormal lower limb bone morphology HP:0040069; Bone fracture HP:0020110

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
